docs/src/playground/components/Sidebar.tsx

```diff
@@ -0,0 +1,167 @@
+import { Box, Divider, Grid, Typography, useMediaQuery, useTheme } from '@mui/material';
+import { Pill } from '@ui-kit-2022/components';
+import React from 'react';
+
+export default function Sidebar() {
+  const theme = useTheme();
+
+  const isTiny = useMediaQuery('(max-width: 375px)');
```

Comment:
Wasn't sure if I should add another breakpoint globally.  So I created a media query based on the smallest breakpoint from the page in Figma.

Perhaps we should consider updating the theme's breakpoints to match Figma?